suite-native/accounts/src/components/AccountListItem.tsx

```diff
@@ -50,15 +58,17 @@ export const AccountListItem = ({ account }: AccountListItemProps) => {
                 </Box>
                 <Text color="gray800">{accountLabel}</Text>
             </Box>
-            <Box alignItems="flex-end">
-                <DiscreetText color="gray1000" typography="hint">
-                    {FiatAmountFormatter.format(fiatAmount ?? 0)}
-                </DiscreetText>
-                <DiscreetText typography="hint" color="gray600">
-                    {CryptoAmountFormatter.format(account.formattedBalance, {
-                        symbol: account.symbol,
-                    })}
-                </DiscreetText>
+            <Box>
+                <Box alignItems="flex-end" justifyContent="flex-end">
+                    <DiscreetText color="gray1000" typography="hint">
```

Comment:
I wouldn't render the discreet text at all if it's testnet. Discreet text renders two elements and might be better to not render it instead of passing it empty string